frontend/src/App.js

```diff
@@ -7,20 +7,22 @@ import { ChakraProvider } from '@chakra-ui/react';
 import { AuthProvider } from './state/AuthProvider';
 import UserProfile from './components/profile';
 import { Navigation } from './components/navigation';
+import DetailComponent from './components/details'
 
 function App() {
+    const storedURL = localStorage.getItem('searchCriteria');
     return (
         <Router>
             <AuthProvider>
                 <ChakraProvider>
                     <div className="App">
-                        <Navigation />
+                        <Navigation searchCriteria={storedURL}/>
                         <Routes>
-                            <Route path='/auto' element={<PlacesAutocompleteComponent />} />
                             <Route path='/search' element={<SearchComponent />} />
-                            <Route path='/auth' element={<UserAuth />} />
```

Comment:
Is there a reason to change this from auth? I kept it general since it can either be the singin/login page